### DISCUSSION TITLE
Schematic and TFT Component Review

### DISCUSSION THREAD

**Discussion by elgranshaman** [Bodmer/TFT_eSPI#1433] [Q&A - General]
Hello,

I am designing a custom board for an ESP32 to drive a [ILI9488 3.5" TFT LCD](https://drive.google.com/file/d/102dUSeXXqQgd_YxoV8GliezGXW2nme9x/view?usp=sharing) using the TFT_eSPI library.

As far as I can tell, I have connected everything correctly and it should work. I have attached a screenshot from the schematic and the datasheet of the display. Kindly review my work and let me know about any red flags.

From the ESP32 side, I've connected the SPI MOSI, SCLK and CS to the respective VSPI pins and DC, RST and BL to GPIOs.

The power supply is 3.3V and it can source up to 1A

Thank you so much!

![Screenshot 2021-11-17 153545](https://user-images.githubusercontent.com/44362719/142166048-7c42154c-0d48-472c-bb8e-8cfcc5fc8a60.png)

**Comment by Bodmer** [comment]
Add a series resistor to the LEDs, 2R2 is a typical value to limit the maximum LED current on 3.3V.

**Reply by elgranshaman** [reply]
Thanks. I plan to control the backlight with the library (PWM) through Q3. In case that doesn't work, I will populate R11

**Reply by Bodmer** [reply]
You still need a series resistor.
